### PR TITLE
Fix date and time formats and add a date formatter filter

### PR DESCRIPTION
**Pull request by RealMattOwen**
Compared all dates and times with the corresponding meetup pages and updated accordingly.

This PR is for https://github.com/leicesterjs/site/issues/37

### CODE REVIEW

**Comment by joshghent** [comment]
Does this need to be updated with the new meetup in #39 ?

File: .eleventy.js [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
+const format = require("date-fns/format");
 
 module.exports = eleventyConfig => {
   eleventyConfig.addPassthroughCopy("src/css");
   eleventyConfig.addPassthroughCopy("src/img");
   eleventyConfig.addPassthroughCopy("src/js");
   eleventyConfig.addPassthroughCopy("src/favicon.ico");
 
+  // Filters
+  eleventyConfig.addNunjucksFilter("formatDate", (startedAt, endedAt) => {
```
Comment (danjordan) [inline .eleventy.js]: Does this work as a universal filter? `eleventyConfig.addFilter`

**Comment by RealMattOwen** [comment]
Note to self - should probably change the README for "Creating a new Meetup"

File: .eleventy.js (hunk 1 above)
Comment (RealMattOwen) [reply .eleventy.js]: Yes it does, that name sounds good to me.. so I've just pushed this change 😁

**Comment by RealMattOwen** [comment]
Would appreciate any feedback on the latest commit 🙂 @danjordan @joshghent

**Pull request merged**